### ISSUE TITLE
go-mp3 codecs

### ISSUE THREAD

**Issue by wsc1** [zikichombo/codec#2]
Incorporate go-mp3.

**Comment by wsc1** [comment]
based on a small C mp3 library.  So we can't incorporate it directly, but perhaps we can make an example people can refer to in case of need while we wait for Go implementations.

**Comment by wsc1** [comment]
Now I'm not sure what made me think it was based on a C mp3 library.  README says pure Go, whatever I saw before didn't.  

go.mod indicates a dependency on oto.

Licensing is Apache; would be nice to see if it's possible to arrive at a single collective license.

**Comment by mewmew** [comment]
> go.mod indicates a dependency on oto.

Would be interesting to see if this dependency can be split out from the rest of the project, to only handle encoding/decoding.

**Comment by wsc1** [comment]
> > go.mod indicates a dependency on oto.
> 
> Would be interesting to see if this dependency can be split out from the rest of the project, to only handle encoding/decoding.

Yes, after [this discussion](https://github.com/hajimehoshi/go-mp3/issues/27), looks like we're going to try to put it in /ext.
